src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -7,29 +7,27 @@
     <DefineConstants>$(DefineConstants);NO_SUPPRESS_GC_TRANSITION</DefineConstants>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
     <IsPackable>true</IsPackable>
+    <EnableConfigurationBindingGenerator>true</EnableConfigurationBindingGenerator>
+    <!-- M.E.Configuration.Binder.SourceGeneration: JavaScriptEncoder not supported; no logic generated which is okay. -->
+    <NoWarn>SYSLIB1100</NoWarn>
+    <!-- Remove after implementing use of Roslyn interceptors feature https://github.com/dotnet/runtime/issues/79527.
+         We want to emit pragma suppressions in emitted code. -->
+    <NoWarn>$(NoWarn);CS0612;CS0618</NoWarn>
```

Comment:
Remove this after fixing https://github.com/dotnet/runtime/issues/88112.

Reply:
Why can't we just do this in the source generated code today? Similar to how the Regex Source Generator does it:

https://github.com/dotnet/runtime/blob/3d6b1ff7e07460213c071e19ca5d37a68779aaf4/src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.cs#L30-L37

Reply:
I think you'll need to fix #88112 to merge this as well.  Seems like that's the cause of 
```
Microsoft.Extensions.Logging.Console.Test.ConsoleLoggerTest.ConsoleLoggerOptions_TimeStampFormat_IsReadFromLoggingConfiguration [FAIL]
      Assert.Equal() Failure
      Expected: yyyyMMddHHmmss
      Actual:   (null)
      Stack Trace:
        /_/src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/ConsoleLoggerTest.cs(1235,0): at Microsoft.Extensions.Logging.Console.Test.ConsoleLoggerTest.ConsoleLoggerOptions_TimeStampFormat_IsReadFromLoggingConfiguration()
           at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
        /_/src/libraries/System.Private.CoreLib/src/System/Reflection/MethodInvoker.cs(59,0): at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

Note the difference in case:
https://github.com/dotnet/runtime/blob/c06d77a74f64d6c5b0280db56fb0c6a43c0aa907/src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/ConsoleLoggerTest.cs#L1224-L1235C60
`TimeStampFormat` vs `TimestampFormat`

Reply:
Yeah that's the plan. I'm planning to include that change in the fix for https://github.com/dotnet/runtime/issues/88112 since they'd both change the baseline files.